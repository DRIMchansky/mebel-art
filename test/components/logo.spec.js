import React from 'react'
import { mount } from 'enzyme'
import renderer from 'react-test-renderer'
import Logo from '../../src/components/ui/logo'

describe('Logo renders correctly', () => {
  it('Without props', () => {
    const tree = renderer.create(<Logo />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('With props', () => {
    const tree = renderer.create(<Logo desctoponly="true" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
