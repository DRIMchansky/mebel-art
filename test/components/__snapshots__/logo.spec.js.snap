// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly With props 1`] = `
.c0 {
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: uppercase;
  font-weight: 300;
  color: #1c5796;
  font-size: 1.3rem;
  -webkit-letter-spacing: -1px;
  -moz-letter-spacing: -1px;
  -ms-letter-spacing: -1px;
  letter-spacing: -1px;
  display: none;
  visibility: hidden;
}

.c0 span {
  font-weight: 800;
  margin-left: 2px;
  color: #283646;
}

@media (min-width:600px) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    visibility: visible;
  }
}

<a
  className="c0"
  desctoponly="true"
  href="/"
>
  Мебель 
  <span>
    Арт
  </span>
</a>
`;

exports[`Logo renders correctly Without props 1`] = `
.c0 {
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: uppercase;
  font-weight: 300;
  color: #1c5796;
  font-size: 1.3rem;
  -webkit-letter-spacing: -1px;
  -moz-letter-spacing: -1px;
  -ms-letter-spacing: -1px;
  letter-spacing: -1px;
}

.c0 span {
  font-weight: 800;
  margin-left: 2px;
  color: #283646;
}

<a
  className="c0"
  href="/"
>
  Мебель 
  <span>
    Арт
  </span>
</a>
`;
